pyuntl/untl_structure.py

```diff
@@ -457,6 +457,40 @@ def generate_form_data(self, **kwargs):
         # Create the form object.
         return FormGenerator(**kwargs)
 
+    def make_hidden(self):
+        """Make an unhidden UNTL element into a hidden element."""
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                # Make the element hidden.
+                if element.content == 'False':
+                    element.content = 'True'
+
+    def make_unhidden(self):
+        """Make a hidden UNTL element into an unhidden element."""
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                # Make the element unhidden.
+                if element.content == 'True':
+                    element.content = 'False'
```

Comment:
I wonder if in these methods we should also account for the possibility of there not being a "meta" tag with qualifier "hidden" and, in that case, create one. In our system I don't think this should really be the case, but not sure...@vphill?

Reply:
I think it would be good to account for the situation where there isn't a meta field with a hidden qualifier.  I imagine those are present in the system somewhere. 

So if there isn't a meta.hidden (meta field with hidden qualifier) then we should add one.

Reply:
Should this be done in all four of the methods? As in, even if we just try to check if a record is hidden, if there is no related tag then we make it? If so we could just create a small method to do so and then call it from all four new methods when needed.

Reply:
Also, just to be clear, are we wanting the default behavior to be that a record is considered visible if it doesn't specify hidden or not?

Reply:
We may want to make the record visible in `make_unhidden()` and make it hidden in `make_hidden()`.

Reply:
Definitely in those, but I was just wondering if it would be desirable to _also_ have a default value added when none is present in the two `is_*` methods

Reply:
I'm not sure, if we'd want to create any meta records since we only want to check if there are any hidden unihidden meta records.

Reply:
Answering just part of the questions that @somexpert asked.

UNTL records should be considered visible if they don't have a meta.hidden field present.

I'm not sure I've thought through the setting of the value if it isn't present.